plugins/chat/app/services/chat/lookup_channel_threads.rb

```diff
@@ -42,6 +42,13 @@ class Contract
       validates :channel_id, presence: true
 
       attribute :limit, :integer
+      validates :limit,
```

Comment:
We should test this :slightly_smiling_face:

Reply:
I did already :p

Reply:
ah you mean in the service?

Reply:
Yep in `plugins/chat/spec/services/chat/lookup_channel_threads_spec.rb` we have a contract spec, it should be pretty straightforward with shoulda matchers :grin: